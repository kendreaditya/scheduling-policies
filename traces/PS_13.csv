1,0,10
2,0,11
3,0,12
4,4,1
5,4,1
6,5,1
7,9,2
8,12,2
9,13,1
10,14,1
11,15,0
12,17,0
13,20,1
14,100,10
15,100,10
16,103,2
17,104,1
18,106,0
19,106,4
20,106,0
21,109,3
22,109,2
23,110,1
24,111,0
25,115,2
26,120,1
27,125,1
28,130,2
29,138,1
30,140,1
