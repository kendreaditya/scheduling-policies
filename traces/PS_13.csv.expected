1,40
2,42
3,43
4,9
5,9
6,9
7,22
8,22
9,16
10,16
11,15
12,17
13,22
14,140
15,140
16,113
17,106
18,106
19,129
20,106
21,125
22,119
23,113
24,111
25,125
26,125
27,129
28,135
29,140
30,141
